read.go

```diff
@@ -65,7 +65,15 @@ func (r *reader) readTag() (*tag.Tag, error) {
 func (r *reader) readVR(isImplicit bool, t tag.Tag) (string, error) {
 	if isImplicit {
 		if entry, err := tag.Find(t); err == nil {
-			return entry.VR, nil
+			switch entry.Tag {
+			case tag.PixelData, tag.OverlayData:
+				// OW takes priority in these cases. See notes at:
+				// 1. https://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_8.1.2
```

Comment:
nit: instead of current, is it possible to perma link to a specific version of the spec in case things get moved around in the future? I know this may get dicey with different parts of the engine linking to different versions, which is also something we should standardize on across our library (e.g. we're targeting this version of the spec).

Reply:
Good idea. Updated to 2024a since 2024b is the current version and doesn't have a link dedicated to it.